modules/module2/module2-06-true_or_false_bars_and_histograms.qmd

```diff
@@ -0,0 +1,261 @@
+---
+format:
+  live-html
+resources:
+  - ../utils.py
+  - ../data
+---
+
+<script src='../../src/quiz.js'></script>  
+
+# 3.1. Exercises
+
+## True or False: Bars and Histograms
+
+<div id='mcq1'></div> 
+<script>
+    generateQuiz(
+        'mcq1', 
+        'True or False',  
+        'It’s often a good idea to make a horizontal histogram so that the labels of the x-axis are easier to read.',  
+        { 
+        'True': 'The labels of histograms are numbers that don’t need to be rotated to be readable.',
+        'False': 'It’s bar charts that can to rotated to make it easier to read the x-axis labels.'
+        },
+        'False',  
+    );
+</script>
+
+<div id='mcq2'></div> 
+<script>
+    generateQuiz(
+        'mcq2',
+        'True or False',
+        'It is best to sort bar charts from high to low values (unless the categorical values have a natural order).',
+        {
+        'True': 'It’s better to sort your bars with some sort of ordering (be it natural or by frequency)!',
+        'False': 'Does sorting a bar chart make it easier to read?'
+        },
+        'True',
+    );
+</script>
+
+## Testing Your Bar Knowledge
+
+<div id='mcq3'></div> 
+<script>
+    generateQuiz(
+        'mcq3', 
+        'Question 1',  
+        'Why do we avoid bar charts when showing summary statistics such as the mean?',  
+        { 
+        'They do not look the best aestetically.': 'Bar plots can be quite useful and beautiful depending on what is communicated!',
+        'Summary statistics are communicated best with line or scatter plots.': 'Is a plot communicating just the summary statistics even a good idea?',
+        'They tend to take too long.': ' Sometimes this can be an issue with coding but not when it comes to visualization.',
+        'They remove a lot of information such as variation amoung the data.': ''
+        },
+        'They remove a lot of information such as variation amoung the data.',  
+    );
+</script>
+
+<div id='mcq4'></div> 
+<script>
+    generateQuiz(
+        'mcq4', 
+        'Question 2',  
+        'If we wanted to increase the number of bars in the histogram (see code below), what would be needed instead of "???" in the code below?',  
+        { 
+        '50': 'You need to use a helper function to increase the number of bars and not just set `bin` equal to a number.',
```

Comment:
```suggestion
        '50': 'You need to use a helper function to increase the number of bars and not just set <code>bin</code> equal to a number.',
```

We need to use HTML syntax directly in the questions, they don't parse markdown. Could you go through and make sure all questions use `<code></code>` instead of backticks?